apps/web/src/components/Basenames/RegistrationBackground/index.tsx

```diff
@@ -39,31 +45,77 @@ export default function RegistrationBackground() {
       </Transition>
       <Transition
         appear
-        show={isClaim || isPending}
-        className={classNames('transition-opacity', registrationTransitionDuration)}
+        show={isClaim}
+        className={classNames(
+          'transition-opacity',
+          registrationTransitionDuration,
+          grayVideoBackgroundClasses,
+        )}
+        enterFrom={classNames('opacity-0')}
+        enterTo={classNames('opacity-100')}
+        leaveFrom="opacity-100"
+        leaveTo="opacity-0"
+      >
+        {/* TODO: Lottie animation file */}
+
+        <Transition
+          appear
+          show={isClaim}
+          className={classNames(
+            'transform-gpu transition-all',
```

Comment:
`transform-gpu` is new to me! v cool

Reply:
yeah it's nice to leverage GPU, tho API for it is not always supported (fallback to cpu)